test/detect/julia/default.txt

```diff
@@ -1,21 +1,5 @@
 ### Types
 
-# Old-style definitions
-
```

Comment:
If we're going to cut all this we need to add in 5-10 lines of code that uses some of the new keywords as well.